dockerfiles/README.md

```diff
@@ -39,6 +40,13 @@ in order to obtain latest bug fixes and updates for the product.
 ##### 4. Build docker images specific to each profile.
 - For integrator, navigate to `<DOCKERFILE_HOME>/integrator` directory. <br>
   Execute `docker build` command as shown below. 
+    + Download [Andes Client](http://maven.wso2.org/nexus/content/groups/wso2-public/org/wso2/andes/wso2/andes-client/3.2.19/) JAR v3.2.19,
+           [Geronimo JMS Spec](http://maven.wso2.org/nexus/content/groups/wso2-public/org/apache/geronimo/specs/wso2/geronimo-jms_1.1_spec/1.1.0.wso2v1/) JAR v1.1.0.wso2v1 and
+           [Secure-vault](http://maven.wso2.org/nexus/content/groups/wso2-public/org/wso2/securevault/org.wso2.securevault/1.0.0-wso2v2/) JAR v.1.0.0-wso2v2 <br>
+           to 
+           ```
+           [docker-ei]/dockerfiles/integrator/lib
```

Comment:
Shall we also change the lib folder to files here, so that it will be consistent, WDYT?

Reply:
I'll make the change.